mpas_analysis/ocean/meridional_heat_transport.py

```diff
@@ -0,0 +1,218 @@
+                      'This likely indicates that the simulation you are analyzing was run with an\n'
+                      'older version of MPAS-O that did not support this flag.  Assuming .false.')
+        mhtAnalysisMemberFlag = '.false.'
+
+    ########################################################################
+    # Mark P Note: Currently only supports global MHT.
+    # Need to add variables merHeatTransLatRegion and merHeatTransLatZRegion
+    # these are not computed by default in ACME right now.
+    # Then we will need to add another section for regions with a loop
+    # over number of regions.
+    ########################################################################
+    variableList = ['avgMeridionalHeatTransportLat',
+                    'avgMeridionalHeatTransportLatZ']
+
+    print '\n  Compute and plot meridional heat content climatology.'
+    simulationStartTime = get_simulation_start_time(runStreams)
+    outputDirectory = build_config_full_path(config, 'output',
+                                             'mpasClimatologySubdirectory')
+    try:
+        os.makedirs(outputDirectory)
+    except OSError:
+        pass
+    outputFileClimo = '{}/meridionalHeatTransport_years{:04d}-{:04d}.nc'.format(
+                       outputDirectory, dictClimo['startYearClimo'],
+                       dictClimo['endYearClimo'])
+    if not os.path.exists(outputFileClimo):
+        print '   Load data...'
+        ds = open_multifile_dataset(
+            fileNames=dictClimo['inputFilesClimo'],
+            calendar=calendar,
+            config=config,
+            simulationStartTime=simulationStartTime,
+            timeVariableName='Time',
+            variableList=variableList,
+            variableMap=variableMap,
+            startDate=dictClimo['startDateClimo'],
+            endDate=dictClimo['endDateClimo'])
+
+        changed, startYear, endYear = \
+            climatology.update_start_end_year(ds, config, calendar)
+        if changed:
+            # update the file name in case the start and end years changed
+            outputFileClimo = \
+                '{}/meridionalHeatTransport_years{:04d}-{:04d}.nc'.format(
+                    outputDirectory, startYear, endYear)
+
+            dictClimo['startYearClimo'] = startYear
+            dictClimo['endYearClimo'] = endYear
+
+        # Compute annual climatology
+        annualClimatology = \
+            climatology.compute_climatology(ds,
+                                            constants.monthDictionary['ANN'],
+                                            calendar)
+
+        # Convert to numpy arrays 
+        # (can result in a memory error for large array size)
+        meridionalHeatTransportLat  = annualClimatology.avgMeridionalHeatTransportLat.values
+        meridionalHeatTransportLatZ = annualClimatology.avgMeridionalHeatTransportLatZ.values
```

Comment:
done.  @xylar please look at e36218c, I think this is right.  Thank you for your attention to detail here, this is great for efficiency, I had assumed that numpy arrays were required for the plotting routines.  I'm amazed that I can transpose an object with a .T:
`z = annualClimatology.avgMeridionalHeatTransportLatZ.T`
without the .values attribute at the end.